view/frontend/web/js/bolt-api-driven-checkout.js

```diff
@@ -1014,6 +1017,25 @@ define([
             );
         },
 
+        boltSetUpSubscriptionFrequencySelector: function() {
+            // deprecated, to be removed
```

Comment:
Could you please add asana task with explanation here?

Reply:
I'm going to make a couple more changes to storm code, then I can remove this piece here.

Reply:
Good! Let's add link to asana